Update status and removed in process items

macgyver13 opened the pull request.

macgyver13 committed:
Update status and removed in process items

yashrajd commented on README.md:
```diff
@@ -12,10 +12,8 @@ This dashboard tracks the current state of development across the Silent Payment
 |--------|-------------|--------|------|
 | **[Central Hub](https://silentpayments.xyz/)** | Silent Payments | ⚙️ | @sethforprivacy
-| **[Visual Roadmap](diagrams/project_overview.mmd)** | Flowchart of SP ecosystem | ⚙️ | Open
-| **[Wallet UX](diagrams/wallet_ux.mmd)** | SP User Flow | ⚙️ | Open
```
More I think about it, I want to see the Bitcoin Design UX page linked to this table, right at the top.

```
| **Silent payments UX** | [Bitcoin Design Guide page](https://bitcoin.design/guide/how-it-works/silent-payments/) | ✅ | @yashrajd 
```

yashrajd commented on README.md:
```diff
@@ -12,10 +12,8 @@ This dashboard tracks the current state of development across the Silent Payment
 | Project | Description | Status | Lead
 |--------|-------------|--------|------|
 | **[Central Hub](https://silentpayments.xyz/)** | Silent Payments | ⚙️ | @sethforprivacy
```
```suggestion
| **[Central Hub](https://silentpayments.xyz/)** | Silent Payments | ⚙️ | [@sethforprivacy](https://github.com/sethforprivacy)
```

yashrajd commented on README.md:
We should add a lead or link to repo/project here, otherwise remove it. We can also add this to Contribute table where this plus 1)mempool scanning 2) descriptor 3)backup file format etc. can be housed later?

Also where did the row for "Outsourced Scanning Server" go? I don't see the line being removed here...

yashrajd commented on README.md:
The project name sounds misleading to me. Maybe just rename it?
Is this repo in active use? Maybe we can ask @sethforprivacy and even potentially add him or Rafael from Cake as lead?

```suggestion
| **Cake's Esplora fork ** | Heavy index, possible SP support | 🧪 | ? | [Cake fork](https://github.com/cake-tech/blockstream-electrs/tree/cake-update-v1) |
```

yashrajd commented on README.md:
Can we get an update on this? This PR seems is open for 8 months with ~0 activity.

yashrajd commented on README.md:
```diff
@@ -12,10 +12,8 @@ This dashboard tracks the current state of development across the Silent Payment
 | Project | Description | Status | Lead
 |--------|-------------|--------|------|
```
We should link to the BIP here as the top item.

yashrajd commented on README.md:
```diff
@@ -60,14 +58,15 @@ This dashboard tracks the current state of development across the Silent Payment
+| **Light Client Protocol** | Fetch tweaks from server | 🧪 / 🆘 | @setavenger | [Delving Discussion](https://delvingbitcoin.org/t/silent-payments-light-client-protocol/891) |
 | **UX Guidelines** | Reusable address + Bolt12 design | ✅ | [@yashrajd](https://github.com/yashrajd/) | [Bitcoin Design Guide page](https://bitcoin.design/guide/how-it-works/silent-payments/) |
-| **BIP-353** | Human-readable address format | ⚙️ | ? | Future enhancement |
+| **BIP-353** | Human readable address format | ⚙️ | ? | [Design Guide](https://bitcoin.design/guide/how-it-works/human-readable-addresses/) |
```
Add link to BIP.

```suggestion
| **[BIP-353](https://github.com/bitcoin/bips/blob/master/bip-0353.mediawiki)** | Human readable address format | ⚙️ | ? | [Design Guide](https://bitcoin.design/guide/how-it-works/human-readable-addresses/) |
```

yashrajd replied:
Also, I'm not so sure this item is very relevant on the dashboard. There is no interaction necessary between silent payments and bip-353 stuff AFAICT, bip-353 is just a way to find the sp-address IMO. Consider removing it.

yashrajd commented on README.md:
```diff
@@ -60,14 +58,15 @@ This dashboard tracks the current state of development across the Silent Payment
 
 ## 📌 How to Contribute
 
 → Join the [Discord](https://discord.gg/AE4DwDhAPZ)
+
 → Start with an issue tagged `help wanted`
```
Not sure how this workflow is supposed to work. Consider removing it, at least for now...

yashrajd replied:
Also consider moving the Discord link to the top (for easy visibility) with a CTA. e.g. 'Join the Discord server (with link) to interact with the community' or something.

yashrajd commented on README.md:
```diff
@@ -60,14 +58,15 @@ This dashboard tracks the current state of development across the Silent Payment
 |-------|-------------|--------|----------------|-------|
-| **Light Client Protocol** | Fetch tweaks from server | 🧪 / 🆘 | @setavenger (?) | [Delving Discussion](https://delvingbitcoin.org/t/silent-payments-light-client-protocol/891) |
+| **Light Client Protocol** | Fetch tweaks from server | 🧪 / 🆘 | @setavenger | [Delving Discussion](https://delvingbitcoin.org/t/silent-payments-light-client-protocol/891) |
 | **UX Guidelines** | Reusable address + Bolt12 design | ✅ | [@yashrajd](https://github.com/yashrajd/) | [Bitcoin Design Guide page](https://bitcoin.design/guide/how-it-works/silent-payments/) |
```
I suggest removing this line, as it has been suggested to be moved in the table at the top.

```suggestion

```

yashrajd reviewed (changes requested):
PR is very timely, love to see it. Shared many thoughts, but happy to discuss stuff etc. to help you merge it pronto!

macgyver13 committed:
updates

yashrajd commented on README.md:
```diff
@@ -1,4 +1,4 @@
-## Silent Payments Developer Dashboard
+## [Silent Payments](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki) Developer Dashboard
```
```suggestion
## Silent Payments Developer Dashboard
```

yashrajd commented on README.md:
```diff
@@ -11,11 +11,10 @@ This dashboard tracks the current state of development across the Silent Payment
-| **[Wallet UX](diagrams/wallet_ux.mmd)** | SP User Flow | ⚙️ | Open
-| **[Tracker](https://docs.google.com/spreadsheets/d/1dXCiAF37UUDs6Hv8jtdQAqfZG6EpwURwCcTk90qnU8g/edit)** | Development Status | ⚙️ | macgyver
-| **[Roadmap](https://docs.google.com/document/d/1ggtPmJWvPCzSoAw0slX4indRDsbm4reDeMjBXreCAzs/edit?tab=t.0)** | Strategy Overview | ⚙️ | macgyver
+| **[Silent payments UX](https://bitcoin.design/guide/how-it-works/silent-payments/)** | Bitcoin Design Guide page | ✅ | yashrajd 
```
```suggestion
| **[Silent payments UX](https://bitcoin.design/guide/how-it-works/silent-payments/)** | Bitcoin Design Guide page | ✅ | @yashrajd 
```

yashrajd committed:
updates

moved BIP link to table; added github profile links at few places; renamed cake's esplora fork say exactly that

yashrajd committed:
Removing orphan rows

Removing rows; issues have been created to capture their content.

yashrajd reviewed (approved):
- BIP link is added at the top of the top table
- orphan rows have been respawned as issues
- Leads have been tagged with their profile links
- all this based off of macgyver's original commits

All this has been discussed with macgyver.
